### ISSUE TITLE
Is this collection going to support the old collection?

### ISSUE THREAD

**Issue by jimccann-rh** [ansible-collections/ibm.cloud#10]
That I am aware of this is not supported any more:

https://github.com/IBM-Cloud/ansible-collection-ibm/

Is this collection to replace that one? If yes any ETA on supporting using Ansible to create BMS in IBM Classic?

**Comment by SteveMunday62** [comment]
Greetings, do we have a date by when "IBM Power Virtual Server in IBM Cloud" functionality will be provided by "this" collection please?

I ask because with https://github.com/IBM-Cloud/ansible-collection-ibm/ deprecated I'm wondering how else can VSI(s) be created using Ansible.

Many thanks, Steve

**Comment by sean-freeman** [comment]
## Important Note

While https://github.com/IBM-Cloud/ansible-collection-ibm should be deprecated _(as it is built from the Terraform Provider to generate all Ansible Modules, and upon execution uses hidden on-the-fly Terraform files in /var/tmp/ansible/ibmcloud)_ - it is not the priority of IBM to maintain either of these Ansible Collections.

I fixed the other one in 2024, gave the fixes to original dev and got it published on Ansible Galaxy, it is the most up to date. However, all future releases of that Ansible Collection are blocked by https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5966 which destroys the internal generator used to build all the Ansible Modules in that Ansible Collection.

In short, good luck to anyone using Ansible with IBM Cloud. The implementation I created you can find for IBM Cloud VS and IBM Power VS > https://github.com/sap-linuxlab/community.sap_infrastructure/tree/main/roles/sap_vm_provision/tasks/platform_ansible

**Comment by SteveMunday62** [comment]
Sean,

Hi, thanks for the update.

Wow, was it really August 2024!!  In the meantime I've been working on my own (internal) PowerVS Role which gives me the key actions we need when de/provisioning, DLPAR-ing, power cycling, tag management, and Gold Image management so am hoping I'm in a good (enough) place for now at least.

Merry Christmas and a very Happy New Year.

All the best, Steve
